sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryClient.java

```diff
@@ -661,4 +661,17 @@ public VoidResponse deleteFileWithResponse(String fileName, Duration timeout, Co
     public String getShareSnapshotId() {
         return directoryAsyncClient.getShareSnapshotId();
     }
+
+    /**
+     * Get directory path from directory client.
+     *
+     * <p>Get directory path. </p>
+     *
+     * {@codesnippet com.azure.storage.file.directoryClient.getName}
+     *
+     * @return The path of the directory.
+     */
+    public String getName() {
+        return directoryAsyncClient.getName();
```

Comment:
Does this name include the share name? If not we should add `getShareName` to `DirectoryClients` and `FileClients`

Reply:
Done.